bc_obps/registration/models.py

```diff
@@ -1,3 +1,134 @@
+import uuid
 from django.db import models
+from phonenumber_field.modelfields import PhoneNumberField
+from django.contrib.gis.db import models
+
+# this may be helpful: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.FileField.upload_to
+class Document(models.Model):
+    file = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, db_comment="")
```

Comment:
I added blank comments so we remember to add them; haven't added them here since columns are still up for debate